queueConversationControl/queueConversationControl.js

```diff
@@ -0,0 +1,187 @@
+var micc = new Micc('localhost');
+var miccSignalr;
+
+function login() {
+    micc = new Micc($('#miccServer').val());
+    micc.login(
+        $('#username').val(),
+        $('#password').val(),
+        connectToQueueHub
+    );
+}
+
+function connectToQueueHub(data) {
+    miccSignalr = new MiccSignalR(data.miccServer, data.access_token);
+    miccSignalr.start(function () {
+        miccSignalr.createQueueHubProxy();
+        miccSignalr.queueStateChanged(populateQueueStats);
+        miccSignalr.queueConversationChanged(addQueueConversationRows);
+        miccSignalr.queueConversationRemoved(removeQueueConversationRows);
+    });
+}
+
+function populateQueueStats(queueData) {
+    if (queueData === undefined || queueData === null) {
+        console.error('populateQueueStats -', queueData);
+        return;
+    }
+    var matchingQueueRows = $(`[id='${queueData.id}']`);
+    if (!matchingQueueRows) {
+        console.warn('No rows found for queue with id %s', queueData.id);
+    } else {
+        matchingQueueRows.each(function (i, queueRow) {
+            var query = $(queueRow);
+            query.attr('id', queueData.id);
+            query.find('td.name').text(queueData.name);
+            query.find('td.reporting').text(queueData.reporting);
+            query.find('td.mediaType').text(queueData.mediaType);
+            query.find('td.waitingConversations').text(queueData.waitingConversations);
+            query.find('td.longestWaiting').text(queueData.longestWaitingConversationDuration);
+            query.find('td.ewt').text(queueData.estimatedWaitTimeForNewConversations);
+            query.find('td.serviceLevel').text(queueData.serviceLevelPercentageToday);
+            query.find('td.offered').text(queueData.offeredConversationsToday);
+            query.find('td.answered').text(queueData.answeredConversationsToday);
+        });
+    }
+}
+
+function populateQueueConversationStats(conversationData) {
```

Comment:
Conversation data isn't a 'stat' in the traditional sense.  Let's not use that word.  Conversation data is fine for verbiage.